File: backend/src/main/java/com/app/backend/dto/CodigoFuenteDTO.java
package com.app.backend.dto;

import java.time.LocalDate;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class CodigoFuenteDTO {
    private Long id;
    private String contenido;
    private LocalDate fecha;
    private String resultado;
    private String tiempo;
}

File: code-execution-service/app/CodigoFuenteDTO.py
# code-execution-service/app/CodigoFuenteDTO.py
from datetime import date
from typing import Optional
from pydantic import BaseModel

class CodigoFuenteDTO(BaseModel):
    contenido: str
    resultado: str = ""
    tiempo: str = ""
    fecha: Optional[date] = None
    id: Optional[int] = None
